src/pretix/control/templatetags/append_next_url.py

```diff
@@ -0,0 +1,12 @@
+from urllib.parse import urlencode
+
+from django import template
+
+register = template.Library()
+
+
+@register.simple_tag
+def append_next(next_url=None):
+    if next_url and next_url.strip():
+        return f'?next={urlencode({"next": next_url})}'
```

Comment:
Please provide a screenshot to prove that you already test this code.

Reply:
Hi @hongquan, thanks for the review, 
I tested the case when manipulate URL in template 
`<a href="{% url "control:auth.register" %}?next={{ request.GET.next|urlencode }}" class="btn btn-link btn-block">`
But after rework, I missed to test it with template tag, I have removed the 'next=' which will cause the 404 error.

Here is the testing result:

![image](https://github.com/user-attachments/assets/aa7ad4d3-e89a-4e80-9bb6-d321f9e423ca)